src/test/scala/org/broadinstitute/hail/driver/GRMSuite.scala

```diff
@@ -133,7 +132,8 @@ class GRMSuite extends SparkSuite {
     val grmBinFile = tmpDir.createTempFile("test", ".grm.bin")
     val grmNBinFile = tmpDir.createTempFile("test", ".grm.N.bin")
 
-    Prop.check(forAll(VariantSampleMatrix.gen[Genotype](sc, (v: Variant) => Genotype.gen(v).filter(_.isCalled))
+    Prop.check(forAll(VariantSampleMatrix.gen[Genotype](sc,
+      VSMSubGens(tGen = (v: Variant) => Genotype.genRealistic(v).map(g => g.filter(_.isCalled)), vGen = Variant.genPlinkCompatible))
       // plink fails with fewer than 2 samples, no variants
       .filter(vsm => vsm.nSamples > 1 && vsm.nVariants > 0),
```

Comment:
I'm not entirely sure what you mean here.  VSMSubGens is a case class with a bunch of default gens